src/index.js

```diff
@@ -2,9 +2,9 @@ import '../styles/main.scss';
 
 export * from '@opuscapita/react-alerts';
 export * from '@opuscapita/react-cards';
+export * from '@opuscapita/react-confirmation-dialog';
 export * from '@opuscapita/react-icons';
```

Comment:
Why do we need to import external libraries here and add these dependencies to package.json?

Reply:
Actually @skarjalainen and me were wondering the same. @havanki is it ok to remove these?

Reply:
@skarjalainen should remember that these were exported from here so that they can be imported through oc-common-ui. But if these are imported directly from their corresponding repos then these are not needed...

Reply:
Alerts, Cards and Icons are already imported through oc-common-ui. So removing export of those means also import fixing. But should I: 1) remove all export (and fix imports), 2) remove only react-confirmation-dialog export or 3) add missing exports (@opuscapita/react-grid)?

Reply:
I'd say that the most correct is to do 1 (remove all exports of opuscapita components and remove those dependencies from package). You are anyway going to update import of ConfirmDialog in OC frontend repos, so it would be logical to update other imports at the same time. It will make future development easier because we won't depend on oc-common-ui version when changing these exported components. It's just my opinion